app/views/shared/_env_for_js.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= json_div(id: 'env-for-js', 'data-env-for-js-json': EnvForJs.new.to_json) %>
+
+<script type="text/javascript">
+const el = document.getElementById('env-for-js');
+const jsonString = el.getAttribute('data-env-for-js-json');
+window.ENV_FOR_JS = JSON.parse(jsonString);
```

Comment:
This works differently than the `serialized_data` template because it needs to load before any other JS, and jQuery isn't loaded at this point.